docker-compose.dev.yml

```diff
@@ -45,6 +53,30 @@ services:
     networks:
       - network
 
+  # keycloak:
```

Comment:
this docker-compose is only for local dev so should we uncomment this now?

Reply:
only local but we can’t test BCeID and IDIR